src/libraries/System.Diagnostics.Process/src/System/Diagnostics/Process.Windows.cs

```diff
@@ -645,6 +648,8 @@ private unsafe bool StartWithCreateProcess(ProcessStartInfo startInfo)
                 _standardError = new StreamReader(new FileStream(parentErrorPipeHandle!, FileAccess.Read, 4096, false), enc, true, 4096);
             }
 
+            commandLine.Dispose();
```

Comment:
Why not defined with a `using` statement?

Reply:
We've generally not added try/finallys just for returning buffers to the array pool.  I'd be fine either way, though.